### PR TITLE
feat(relocation): Add auto redirect after 2FA enroll

### PR DESCRIPTION
**Pull request by azaslavsky**

### CODE REVIEW

File: static/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx [hunk 1]
```diff
@@ -330,6 +335,36 @@ class AccountSecurityEnroll extends DeprecatedAsyncView<Props, State> {
...
+    // (this is usually only the case for a newly claimed relocated user), we redirect to the org
+    // slug's subdomain now.
+    const subDomains = getSubDomains(new URL(window.location.href));
+    const firstSubDomain = subDomains.length > 0 ? subDomains[0] : '';
+    const isAlreadyInOrgSubDomain = orgs.some(org => {
+      return org.slug === firstSubDomain;
+    });
+    if (!isAlreadyInOrgSubDomain) {
+      window.location.assign(generateOrgSlugUrl(orgs[0].slug));
+    }
```
Comment (markstory) [inline static/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx:367]: Organizations from the API should include a `links` property that has both `organizationUrl` and `regionUrl` for the organization. You could use those values to compare with the current host and skip splitting up subdomains.

File: static/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx [hunk 2]
```diff
@@ -330,6 +335,36 @@ class AccountSecurityEnroll extends DeprecatedAsyncView<Props, State> {
 
     this.props.router.push('/settings/account/security/');
     openRecoveryOptions({authenticatorName: this.authenticatorName});
+
+    // The remainder of this function is included primarily to smooth out the relocation flow. The
+    // newly claimed user will have landed on `https://sentry.sentry.io/settings/account/security`
```
[the remark this answers is not in the document]
Comment (azaslavsky) [reply static/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx:340]: Fixed this comment - meant to say it actually goes to `sentry.io/settings/...`, which I've confirmed is the case.

File: static/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx (hunk 1 above)
Comment (azaslavsky) [reply static/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx:367]: Switched to using `organizationUrl` as you suggest in the latest push.

**Review by hubertdeng123 — APPROVED** [review]

**Review by markstory — COMMENTED** [review]
Looks good.

**Pull request merged**